### ISSUE TITLE
Can NOT reproduce results on MOT20

### ISSUE THREAD

**Issue by HaojunYuPKU** [PeizeSun/TransTrack#75]
Hello, I'm trying to reproduce results on MOT20, but I got a 5 points drop using the default hyper-parameter from MOT17. Can you provide the training script on MOT20?

**Comment by tran7777777** [comment]
Hello, have you reproduced the mot20? I also encountered this problem, and the accuracy has dropped a lot after reproduction. If you reproduce successfully, would it be convenient to send your parameters? Thank you very much.
